### ISSUE TITLE
Just a few questions

### ISSUE THREAD

**Issue by rottenpants466** [dmitry-s93/MControlCenter#61]
Is there an option to switch automatically between power modes in MControlCenter?

For example, on my system preferences in kubuntu i have 2 power mode profiles:

- When connected to AC i choose performance mode
- When only using battery choose power save

MControlCenter doesn't change automatically between power modes if i connect the AC charger or remove it, i have to do it manually. Also, if i'm using the performance mode in system settings and super battery in McontrolCenter the laptop runs really hot because the fans are turned off.

Also is it possible to automatically change the screen refresh rate between power modes? For example, 165Hz with AC connected and 60Hz when only using the battery?

Thank you

**Comment by dmitry-s93** [comment]
> Is there an option to switch automatically between power modes in MControlCenter?

No, but this can be added in the future.

> Also is it possible to automatically change the screen refresh rate between power modes? For example, 165Hz with AC connected and 60Hz when only using the battery?

Not planned yet.

**Comment by rottenpants466** [comment]
> > Is there an option to switch automatically between power modes in MControlCenter?
> 
> No, but this can be added in the future.
> 
> > Also is it possible to automatically change the screen refresh rate between power modes? For example, 165Hz with AC connected and 60Hz when only using the battery?
> 
> Not planned yet.

cheers

**Comment by teackot** [comment]
> For example, 165Hz with AC connected and 60Hz when only using the battery?

You can probably write a udev rule: https://unix.stackexchange.com/a/321921

**Comment by Reginald-Gillespie** [comment]
I agree with power mode switching - all I'd need for this is just for the tool to support command-line options, that should be easier. Currently my system automatically runs a script when it is plugged-in / unplugged, which uses the command line of cpupower-gui to set the CPUs into a performance mode while plugged in, it would be nice if this script supported command line options so we can do things like that. 

Other than that, I love that this has been implemented into linux, specifically the cooler boost is why I installed it, thanks!

**Comment by nitro122** [comment]
> Is there an option to switch automatically between power modes in MControlCenter?
> 
> For example, on my system preferences in kubuntu i have 2 power mode profiles:
> 
>     * When connected to AC i choose performance mode
> 
>     * When only using battery choose power save
> 
> 
> MControlCenter doesn't change automatically between power modes if i connect the AC charger or remove it, i have to do it manually. Also, if i'm using the performance mode in system settings and super battery in McontrolCenter the laptop runs really hot because the fans are turned off.
> 
> Also is it possible to automatically change the screen refresh rate between power modes? For example, 165Hz with AC connected and 60Hz when only using the battery?
> 
> Thank you

u can save a udev command to this. It works quite flawlessly

**Comment by mutchiko** [comment]
you can track progress in this issue: https://github.com/dmitry-s93/MControlCenter/issues/249
